readthedocs/rtd_tests/tests/test_builds.py

```diff
@@ -116,9 +124,8 @@ def test_build_respects_yaml(self):
 
         build_env = LocalBuildEnvironment(project=project, version=version, build={})
         python_env = Virtualenv(version=version, build_env=build_env)
-        config = ConfigWrapper(version=version, yaml_config=create_load({
-            'formats': ['epub']
-        })()[0])
+
+        config = load_yaml_config(version)
```

Comment:
If we create the ConfigWrapper object directly, we don't call to `validate` and the default values are never applied.